### DISCUSSION TITLE
`icon` Prop for Buttons

### DISCUSSION THREAD

**Discussion by SDIjeremy** [formkit/formkit#755] [Ideas]
When we just want to have a button with no (visible) label but an icon instead...
It would be nice to do `<FormKit type="button" icon="bitcoin" />` and have just the icon and also an accessibility label.

This prevents us from having to do any weird `sections-schema` or css to fix a prefix icon.

**Comment by anbraten** [comment]
A pure icon button would be really helpful. I initially expected this to be achievable by providing an empty default slot or empty label and prefix-icon. Closest I could get without "hacking" css was using `<FormKit type="button" prefix-icon="trash" label=" " />`

**Comment by fenilli** [comment]
Well for sure this seens like a good idea, but also it is really easy to export the button and just change it so it accepts an icon as label for example, so you could name it `button-icon`.
